{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oop", "trace": "def test_broken():\n>       raise Exception('oop')\nE       Exception: oop\n\ntest_alluredemo.py:12: Exception"}, "start": 1637912136847, "stop": 1637912136847, "uuid": "33ed92de-943d-43c6-947f-19475f13c6be", "historyId": "fb799eecaee64eb20ac70a0047ddd7ff", "testCaseId": "54327feac6bee8ccd45a88d670bf2886", "fullName": "test_alluredemo#test_broken", "labels": [{"name": "suite", "value": "test_alluredemo"}, {"name": "host", "value": "LAPTOP-QMS0DV6U"}, {"name": "thread", "value": "21592-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_alluredemo"}]}